core/internal/filetransfer/file_transfer_manager.go

```diff
@@ -176,11 +182,16 @@ func (fm *fileTransferManager) Close() {
 // Uploads or downloads a file.
 func (fm *fileTransferManager) transfer(task *Task) error {
 	var err error
+	// TODO: hardcoding to use default file transfer until more transfers are implemented
+	fileTransfer := fm.fileTransfers[FileTransferTypeDefault]
+	if fileTransfer == nil {
+		return fmt.Errorf("file transfer default not found")
+	}
```

Comment:
code style: the `var err error` line should stay attached to the `switch` statement since it's not relevant to this new block